.travis.yml

```diff
@@ -1,12 +1,24 @@
+language: python
+
+python:
+  - "3.4"
+  - "3.6"
+  - "3.7-dev"
+
+install:
+ - python -m pip install flake8
```

Comment:
I don't know if it makes a difference, but I'd use python3.

Reply:
Each run is done in separate virtualenv instances and bin/python is linked to the correct version.

Reply:
Ah, sure. `python-flake8` was the one that confused me.

Reply:
Ooh, forgot to drop python-flake8 from the commit. Will do before merging.

When we re-enable the actual build we will can change to using python3 and drop setting the environment variable :smile: